rov/sensors/README.md

```diff
@@ -25,22 +26,24 @@ class Sensor(object):
         self._data['pressure'] += 5
 ```
 
-The advantage to this is that all sensors can be updated quickly by calling update,
-and data can then be extracted separately when needed.
+The advantage to this is that all sensors can be updated quickly by calling
+update, and data can then be extracted separately when needed.
 
-Additionally, for each sensor, a test script should be provided to easily verify that it
-is working. Lastly, a mock library should be made in place in case the sensor is not working.
-This lets the ROV still run while a sensor may not be connected. For example, here is how to handle
-dynamically choosing the mock library for the pressure sensor:
-```python
-from BlueRobotics_Bar30 import Pressure as BlueRobotics_Bar30
-from Pressure_Mock import Pressure as Pressure_Mock
+Additionally, for each sensor, a test script should be provided to easily
+verify that it is working.
 
+## Detecting Failed Imports and Using Mock Library
+Lastly, a mock library should be made in place in case the sensor is not
+working.  This lets the ROV still run while a sensor may not be connected. For
+example, here is how to handle dynamically choosing the mock library for the
+pressure sensor:
+```python
```

Comment:
Ah, I didn't realize those were real imports. I thought they were a part of the documentation.